src/constants/constants.ts

```diff
@@ -1,2 +1,25 @@
+import { SwapperName } from 'lib/swapper/api'
+import { assertUnreachable } from 'lib/utils'
+
 export const USDC_PRECISION = 6
-export const DEFAULT_SLIPPAGE = '0.002' // .2%
+
+// Slippage defaults. Don't export these to ensure the getDefaultSlippagePercentageForSwapper helper function is used.
+const DEFAULT_SLIPPAGE_DECIMAL_PERCENTAGE = '0.002' // .2%
+const DEFAULT_COWSWAP_SLIPPAGE_DECIMAL_PERCENTAGE = '0.005' // .5%
+
+export const getDefaultSlippagePercentageForSwapper = (swapperName?: SwapperName): string => {
```

Comment:
its weird that swapperName is optional, is there ever a case where a swapper doesnt have a name?